twilio/http/http_client.py

```diff
@@ -78,6 +78,11 @@ def request(
         elif timeout <= 0:
             raise ValueError(timeout)
 
+        if headers:
+            if "Requires-Authentication" in headers:
+                headers.pop("Requires-Authentication", None)
```

Comment:
check with kridai if sending auth information in auth header for no auth apis is fine - we will get a success reposne in this case but auth headers will be sent

Reply:
this is removed now